rtb/analisi_dei_requisiti_ver0.4.0.typ

```diff
@@ -0,0 +1,732 @@
+- *Trigger*: l'amministratore desidera visualizzare il nome del punto di interesse collegato all'annuncio.
+- *Scenario principale*:
+  + L'amministratore accede alla sezione apposita degli annunci.
+  + Viene esposto il nome del punto di interesse collegato all'annuncio tra le informazioni principali del singolo annuncio.
+  + Viene esposto il nome del punto di interesse collegato all'annuncio tra i dettagli dell'annuncio.
+
+=== UC21 - Visualizzazione _e-mail_ dell'utente destinatario dell'annuncio <uc21>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore visualizza le informazioni principali di un annuncio nello storico o ha selezionato i dettagli dell'annuncio.
+- *Postcondizioni*: l'amministratore visualizza l'_e-mail_ dell'untente destinatario dell'annuncio selezionato.
+- *Trigger*: l'amministratore desidera visualizzare l'_e-mail_ dell'untente destinatario dell'annuncio.
+- *Scenario principale*:
+  + L'amministratore accede alla sezione apposita degli annunci.
+  + Viene esposta l'_e-mail_ dell'untente destinatario dell'annuncio tra le informazioni principali del singolo annuncio.
+  + Viene esposta l'_e-mail_ dell'untente destinatario dell'annuncio tra i dettagli dell'annuncio.
+
+=== UC22 - Visualizzazione data e ora di emissione dell'annuncio <uc22>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore visualizza le informazioni principali di un annuncio nello storico o ha selezionato i dettagli dell'annuncio.
+- *Postcondizioni*: l'amministratore visualizza la data e l'ora di emissione dell'annuncio selezionato.
+- *Trigger*: l'amministratore desidera visualizzare la data e l'ora di erogazione dell'annuncio.
+- *Scenario principale*:
+  + L'amministratore accede alla sezione apposita degli annunci.
+  + Viene esposta la data e l'ora di emissione dell'annuncio tra le informazioni principali del singolo annuncio.
+  + Viene esposta la data e l'ora di erogazione dell'annuncio tra i dettagli dell'annuncio.
+
+=== UC23 - Visualizzazione _flag_ sugli annunci nello storico inviati a noleggi attivi
+<uc23>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore visualizza le informazioni principali di un annuncio nello storico o ha selezionato i dettagli dell'annuncio.
+- *Postcondizioni*: l'amministratore visualizza un _flag_ sugli annunci inviati a noleggi che sono ancora attivi.
+- *Trigger*: l'amministratore desidera visualizzare se l'annuncio è stato inviato ad un mezzo con noleggio attivo.
+- *Scenario principale*:
+  + L'amministratore accede alla sezione apposita degli annunci.
+  + Viene esposto un _flag_ tra le informazioni principali del singolo annuncio che indica se il noleggio è attivo in quel momento.
+  + Viene espost un _flag_ tra i dettagli dell'annuncio che indica se il noleggio è attivo in quel momento.
+#v(20pt)
+#figure(
+  image("../assets/use_cases/storico.png"),
```

Comment:
ho un dubbio sul fatto che singolo annuncio sia sotto a storico, provo a spiegare il perché (in caso ci campiamo meglio a voce):
Cardin ha detto che i sottocasi si hanno quando si è vincolati ad averli, ad esempio "data e ora emissione" per forza sarà dentro a "singolo annuncio"
Il fatto però che si possa visualizzare il singolo annuncio solo dopo aver acceduto alla lista è forse una scelta di design della GUI, quindi un vincolo che in realtà non esisterebbe e quindi UC indipendente